client/src/apollo/apollo-router-guards.js

```diff
@@ -53,6 +53,72 @@ export async function beforeEachRequiresSubmissionAccess(
   }
 }
 
+export async function beforeEachRequiresReviewAccess(
```

Comment:
I think there's rework we need to do here with how we're checking for access by role + if the submission is open.  Some of this should perhaps be relocated to a mounted hook on the component, but we need to figure out where the separation of concerns should lie between router hooks and the mounted hook on the component.

(Just a note for future, it works now, I think its just going to get messier as the application progresses if we don't address it though.)